layout/templates/key_stories.mustache

```diff
@@ -1,65 +1,92 @@
 <div id="key-stories" class="viz-text">
 
-  <h2>Key Stories</h2>
+  <h2>How do we use water in the U.S.?</h2>
```

Comment:
My take on it is that both are fine in the long run. I  think editing the template gives us a bit more flexibility easily...but I don't think there's anything we couldn't do via markdown that we do directly in the template too. 

That being said, the reason I went here with the template is because Marty added the key-story templates this way. He's going to probably be the one doing the bulk of the styling, so if he likes it better this way, that's totally fine by me.

My rule of thumb is: if I know the text is going to be pretty vanilla (ie...a text description of how to use the app...that's not going to require a lot of format/style), I prefer markdown.  But texts with lists and images and who-knows-what...that's going to be something we'll want some real fine control over...let's go straight to a custom template

Reply:
I don't have a strong preference either way. I agree with Laura - I'm comfortable with choosing what's easiest for the situation, so 👍